packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -110,6 +110,7 @@ void _updateGeneratedEnvironmentVariablesScript({
   for (final String line in xcodeBuildSettings) {
     localsBuffer.writeln('export "$line"');
   }
```

Comment:
can we add a comment or rename the private method to make it harder to accidentally refactor the normal flow to use this script and get verbose? e.g. make the verbose <-> script <-> add-to-app linkage more explicit?

Reply:
You're right, I can move this to a more explicit add-to-app spot.

Reply:
Moved it into the pod script phase.